src/stylesheets/core/_utilities.scss

```diff
@@ -26,22 +26,12 @@
 // Unstyled list helper
 
 @mixin unstyled-list() {
-  display: block;
+  @include margin(0 null);
```

Comment:
Done in 46a01a7. This will make it more consistent across browsers and match our paragraph text margins.